graze/pistachio
# Upload stylestats results to S3, making them public.

**changes**
- Removes unused js for generating css statistics, including its associated dependency 'moment'.
- No longer includes the css path or date/time when generating css statistics with the gulp command `build:stats`.
- Changes the directory for generated css statistics from `tests/results` to `statistics/css`.
- When publishing a new version of Pistachio css statistics are also published, making them publicly accessible.
- Configures Travis CI to use the latest 4.x release of node.js, which was required by our stylestats dependancy.

**qa**
- Confirm references to `tests/results` are updated correctly.
- Confirm `site/lib/css-stats.js` and its dependancy `moment` is indeed unused.
- Check dev build and deploy work as one would expected. I've been able to run the publish command locally without issue (see below).
- Agree there's no reason these should not be publicly available.

**next steps**
- Generate and publish historical stats.
- Consume statistics with Sumologic for internal visualisations.

**example**
https://pistachio-cdn.graze.com/dev/statistics/css/pistachio.json
